test/integration/test_navbar_links.js

```diff
@@ -0,0 +1,139 @@
+/*
+ * Checks that the links in the navbar on the homepage have the right URLs to redirect to
+ *
+ * Test cases: https://github.com/LLK/scratch-www/wiki/Most-Important-Workflows
+ */
+
+require('chromedriver');
```

Comment:
Is this line here necessary? `chromedriver` is not in the requirements, so I got errors when I tried to run these tests.